cli/tests/pcluster/schemas/test_schema_validators.py

```diff
@@ -154,8 +154,16 @@ def test_awsbatch_compute_resource_validator(section_dict, expected_message):
     "section_dict, expected_message",
     [
         ({"ComputeResources": []}, "Length must be 1"),
-        ({"ComputeResources": [{"InstanceType": "c5.xlarge"}]}, None),
-        ({"ComputeResources": [{"InstanceType": "c5.xlarge"}, {"InstanceType": "c4.xlarge"}]}, "Length must be 1"),
+        ({"ComputeResources": [{"Name": "compute_resource1", "InstanceType": "c5.xlarge"}]}, None),
+        (
+            {
+                "ComputeResources": [
+                    {"Name": "compute_resource1", "InstanceType": "c5.xlarge"},
```

Comment:
I think we should re-use the QueueNameValidator for the compute name too or put some code in common.

I don't think we can have any value for the compute resource name.

Reply:
Done, any reason to refuse "default" as a name?